util/web.go

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package util
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"runtime"
+)
+
+const (
+	defaultServer = "http://localhost:3000/"
+)
+
+//Runs the frontend/backend for OAuth2l Playground
+func Web() {
+	_, err := os.Stat("/oauth2l-web")
+	if os.IsNotExist(err) {
+		fmt.Println("This feature is not currently installed. Installing...")
+		cmd := exec.Command("git", "clone", "https://github.com/googleinterns/oauth2l-web.git", "oauth2l-web")
```

Comment:
I would clone this into an absolute path, say ~/.oauth2l-web, and provide an option to override the installation directory. The reason is that oauth2l can be launched from bin/ or some highly privileged location, and it may be inappropriate to clone a git repo there.

Reply:
I implemented the suggestion in the new push.